src/main/java/tests/formatTests/ShouldCheckRepeatingCharactersInStates.java

```diff
@@ -0,0 +1,45 @@
+package tests.formatTests;
+
+import domain.pages.DirectoryPage;
+import domain.pages.HomePage;
+import org.junit.Test;
+import org.openqa.selenium.support.PageFactory;
+import tests.uiTests.UITestBaseClass;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import org.apache.commons.collections.bidimap.DualHashBidiMap;
+
+public class ShouldCheckRepeatingCharactersInStates extends UITestBaseClass {
+
+    @Test
+    public void printTheMostRepeatingCharacterInStates() {
+        HomePage homePage = new PageFactory().initElements(driver, HomePage.class);
+        homePage.clickRepresentativesLink();
+        DirectoryPage directoryPage = new PageFactory().initElements(driver, DirectoryPage.class);
+        directoryPage.clickListByStateandDistrict();
+
+
+        Map<String, String> repeatingChars = new HashMap<String, String>();
```

Comment:
OMG

Reply:
I did this to understand how hashmap works. I have no idea before on hashmap. We can practice on this.

Reply:
done